.github/workflows/build.yml

```diff
@@ -15,11 +15,18 @@ jobs:
       with:
         go-version: '1.21'
 
-    - name: Run tests
+    - name: Run all tests
+      if: matrix.pgVersion != '14.8'
       run: go test ./...
       env:
         POSTGRES_VERSION: ${{ matrix.pgVersion }}
 
+    - name: Run pg14 tests
+      if: matrix.pgVersion == '14.8'
+      run: go test ./... --skip 'TestViewsAreCreatedWithSecurityInvokerTrue'
```

Comment:
Could we skip the test from the test case instead? calling `t.Skip` or similar based on the postgres version

Reply:
I've rebased to skip the test as you suggest rather than skipping in github actions.